src/Configuration/ConfigDefinition.php

```diff
@@ -16,8 +16,7 @@ class ConfigDefinition extends BaseConfigDefinition
     public const MODE_RAW = 'raw';
     public const MODE_MAPPING = 'mapping';
 
-    private const QUERY_INCOMPATIBLE_NODES = ['select', 'limit', 'incrementalFetchingKey'];
-    private const INCREMENTAL_FETCHING_INCOMPATIBLE_NODES = ['select'];
+    private const INCREMENTAL_FETCHING_INCOMPATIBLE_NODES = ['select', 'filter', 'limit'];
```

Comment:
- `select` nie je povoleny s inc. fetching -> aby bol inc. f. kluc pritomny vo vysledku
- `filter` -> nie je ..., pretoze inc. f. pouziva vlastny filter (`WHERE`)
- `limit` -> nie je ...., pretoze db nepodporuje `ORDER BY`, a ak das limit `100`, tak nic negarantuje, ze dostanes 100 "najstarsich" zaznamov, ... ale iba `100` nahodnych ... preto musime pri inc. f. stiahnut naraz vsetky nove data, ... tam potom mozeme najst maximum a to si ulozit.